packages/ui/src/app/demos.tsx

```diff
@@ -73,6 +78,30 @@ export class Demos extends Component<
         );
       },
     },
+    [FLAGD_OFREP_PROVIDER_ID]: {
+      factory: () => {
+        const ofrepConfig = this.state.availableProviders.find((p) => p.id === FLAGD_OFREP_PROVIDER_ID);
+        const tls = ofrepConfig?.tls ?? false;
+        const host = ofrepConfig?.host ?? 'localhost';
+        const port = ofrepConfig?.port ?? 8016;
+        const baseUrl = `${tls ? 'https' : 'http'}://${host}:${port}`;
+        return new OFREPWebProvider({ baseUrl }, console);
```

Comment:
```suggestion
        return new OFREPWebProvider({ baseUrl, pollInterval: 1000 }, console);
```

I think we should set the polling interval to be more aggressive. This is for demo purposes, so waiting up to 30 seconds could lead to awkward silence 😄.

Reply:
Should we go to something as little as 10 seconds?